### ISSUE TITLE
No asciidoc support?

### ISSUE THREAD

**Issue by bougyman** [segmentio/terraform-docs#275]
**Describe the bug**

I installed the release of v1.9.1 and there doesn't seem to be asciidoc support. I used
```
 % curl -Lo ~/bin/terraform-docs https://github.com/segmentio/terraform-docs/releases/download/v0.9.1/terraform-docs-v0.9.1-$(uname | tr '[:upper:]' '[:lower:]')-amd64                                                                                                                                  
chmod +x ~/bin/terraform-docs
```

To install, and get an error when trying the asciidoc command (non-zero exit), and it just shows the help. The help also doesn't list asciidoc support:

```
$ terraform-docs --version
terraform-docs version v0.9.1 7f761d3 linux/amd64 BuildDate: 2020-04-02T21:58:38+0000                                                             
tvanderpoel@bougypad ~/g/terraform_repo                                                                                                           
 % terraform-docs asciidoc ./tst_module                                                                                                                   
A utility to generate documentation from Terraform modules in various output formats                                                              

Usage:
  terraform-docs [command]

Available Commands:
  completion  Generate autocomplete for terraform-docs
  help        Help about any command
  json        Generate JSON of inputs and outputs
  markdown    Generate Markdown of inputs and outputs
  pretty      Generate colorized pretty of inputs and outputs
  tfvars      Generate terraform.tfvars of inputs
  version     Print the version number of terraform-docs
  xml         Generate XML of inputs and outputs
  yaml        Generate YAML of inputs and outputs

Flags:
      --header-from string             relative path of a file to read header from (default "main.tf")
  -h, --help                           help for terraform-docs
      --no-header                      do not show module header
      --no-inputs                      do not show inputs
      --no-outputs                     do not show outputs
      --no-providers                   do not show providers
      --no-requirements                do not show module requirements
      --no-sort                        do no sort items
      --output-values                  inject output values into outputs
      --output-values-from string      inject output values from file into outputs
      --sort-by-required               sort items by name and print required ones first
      --sort-inputs-by-required        [deprecated] use '--sort-by-required' instead
  -v, --version                        version for terraform-docs
      --with-aggregate-type-defaults   [deprecated] print default values of aggregate types

Use "terraform-docs [command] --help" for more information about a command.
```
**To Reproduce**

1. Install with curl
2. Run `terraform-docs asciidoc ./some_module`
3. The output will simply be the help text from above

**Expected behavior**

Expected the asciidoc command to be available as a command and in the help

**Version information:**

- terraform-docs version (use `terraform-docs --version`): 1.9.1
- Go Version: Did not manually build
- OS: Ubuntu 18.04

**Comment by khos2ow** [comment]
Support for asciidoc was added in #241 and will be available in the next release, `v0.10.0`.

**Comment by bougyman** [comment]
Ah, ok. That makes sense, thanks.
